core/app/core/src/lib/fields/field-logic/field-logic.manager.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * SuiteCRM is a customer relationship management program developed by SalesAgility Ltd.
- * Copyright (C) 2021 SalesAgility Ltd.
+ * Copyright (C) 2023 SalesAgility Ltd.
```

Comment:
Hi @mpuyosa91 this should probably still be `2021` which is when the file was added

Reply:
Done